diff --git a/test/onlinecheck.c b/test/onlinecheck.c
index 787a130..4999da6 100644
--- a/test/onlinecheck.c
+++ b/test/onlinecheck.c
@@ -57,6 +57,8 @@ void readcert (void) {
  * case when we decide to look at the client name after all.
  */
 void readpgpkey (void) {
+	printf ("Note: PGP was removed from TLS 1.3 and the keys are not read\n");
+	return;
 	int ok = 1;
 	int fd = open ("tlspool-test-client-pubkey.pgp", O_RDONLY);
 	ok = ok && (fd >= 0);
diff --git a/testdata/Makefile b/testdata/Makefile
index 00ba757..2142a60 100644
--- a/testdata/Makefile
+++ b/testdata/Makefile
@@ -171,7 +171,9 @@ fill-cert: $(TARGET_CERT)
 clean-pgp:
 	rm -f $(TARGET_PGP)
 
-fill-pgp: $(TARGET_PGP)
+# fill-pgp: $(TARGET_PGP)
+fill-pgp:
+	echo "NOTE: PGP is not part of TLS 1.3 anymore and has been removed from the testdata"
 
 clean-db:
 	if pidof tlspool ; then echo First stop TLS Pool ; exit 1 ; fi
@@ -314,8 +316,8 @@ tlspool.env:
 	chown $(DMNUSR):$(DMNGRP) $@
 
 localid.db: tlspool.env
-	$(TOOLDIR)/tlspool-localid-set $(CONFFILE) testcli@tlspool.arpa2.lab OpenPGP,client '$(PRIVKEY1)' tlspool-test-client-pubkey.pgp
-	$(TOOLDIR)/tlspool-localid-set $(CONFFILE) testsrv@tlspool.arpa2.lab OpenPGP,server '$(PRIVKEY2)' tlspool-test-server-pubkey.pgp
+	#DROPPED-IN-TLS-1.3# $(TOOLDIR)/tlspool-localid-set $(CONFFILE) testcli@tlspool.arpa2.lab OpenPGP,client '$(PRIVKEY1)' tlspool-test-client-pubkey.pgp
+	#DROPPED-IN-TLS-1.3# $(TOOLDIR)/tlspool-localid-set $(CONFFILE) testsrv@tlspool.arpa2.lab OpenPGP,server '$(PRIVKEY2)' tlspool-test-server-pubkey.pgp
 	$(TOOLDIR)/tlspool-localid-set $(CONFFILE) testcli@tlspool.arpa2.lab x.509,client '$(PRIVKEY3)' tlspool-test-client-cert.der
 	$(TOOLDIR)/tlspool-localid-set $(CONFFILE) testsrv@tlspool.arpa2.lab x.509,server '$(PRIVKEY4)' tlspool-test-server-cert.der
 	$(TOOLDIR)/tlspool-localid-set $(CONFFILE) testcli@tlspool.arpa2.lab kerberos,client,server 'pkcs11:some;place' /dev/null
@@ -334,8 +336,8 @@ disclose.db: tlspool.env localid.db
 	chown $(DMNUSR):$(DMNGRP) $(BDBENV)/* $@
 
 trust.db: tlspool.env tlspool-test-ca-cert.der tlspool-test-ca-cert.keyid tlspool-test-flying-signer.der tlspool-test-flying-signer.keyid
-	$(TOOLDIR)/tlspool-trust-set $(CONFFILE) x509,client,server `cat tlspool-test-ca-cert.keyid` 1 tlspool-test-ca-cert.der
-	$(TOOLDIR)/tlspool-trust-set $(CONFFILE) x509,client,server `cat tlspool-test-flying-signer.keyid` 1 tlspool-test-flying-signer.der
+	$(TOOLDIR)/tlspool-trust-set $(CONFFILE) x509,client,server `cat tlspool-test-ca-cert.keyid | head -n 1` 1 tlspool-test-ca-cert.der
+	$(TOOLDIR)/tlspool-trust-set $(CONFFILE) x509,client,server `cat tlspool-test-flying-signer.keyid | head -n 1` 1 tlspool-test-flying-signer.der
 	chown $(DMNUSR):$(DMNGRP) $(BDBENV)/* $@
 
 clean: clean-db
